mainwindow.ui

```diff
@@ -52,7 +55,7 @@
             <string>Launch</string>
            </property>
            <property name="icon">
-            <iconset resource="qtresources.qrc">
+            <iconset theme="system-run" resource="qtresources.qrc">
```

Comment:
good thing applications aren’t webpages, and the user can expect a common design language and integration with desktop environment features,

especially for small tools like this one, that’s valuable. nobody likes to be bombarded with new icon sets, widget design, and color schemes for every small task.